pkg/operator/ceph/object/spec.go

```diff
@@ -314,42 +328,34 @@ func (c *clusterConfig) makeDaemonContainer(rgwConfig *rgwConfig) v1.Container {
 	}
 	kmsEnabled, err := c.CheckRGWKMS()
 	if err != nil {
-		logger.Errorf("failed to enable KMS. %v", err)
+		logger.Errorf("failed to enable SSE-KMS. %v", err)
```

Comment:
IMO, we really should be failing if this fails to enable. This means that a feature the user is intending to enable won't work, which most users would consider an error.